supplementary/alica_ros1/alica_tests/src/test/test_blackboard.cpp

```diff
@@ -150,49 +148,49 @@ TEST_F(TestBlackboard, testWithoutPlan)
 TEST_F(TestBlackboard, testMappingFromBool)
 {
     EXPECT_TRUE((checkMapping<bool, bool>(BBType::BOOL, true, BBType::BOOL, true)));
-    EXPECT_THROW((checkMapping<bool, int64_t>(BBType::BOOL, true, BBType::INT64, 1)), BlackboardException);
-    EXPECT_THROW((checkMapping<bool, uint64_t>(BBType::BOOL, true, BBType::UINT64, 1u)), BlackboardException);
-    EXPECT_THROW((checkMapping<bool, double>(BBType::BOOL, true, BBType::DOUBLE, 1.0)), BlackboardException);
-    EXPECT_THROW((checkMapping<bool, std::string>(BBType::BOOL, true, BBType::STRING, "true")), BlackboardException);
+    EXPECT_THROW((checkMapping<bool, int64_t>(BBType::BOOL, true, BBType::INT64, 1)), internal::BlackboardException);
```

Comment:
I don't think the exception should be in internal namespace

Reply:
I also think the exception should not be internal, because it is part of how a user of the blackboard interacts with the blackboard, by receiving and catching these exceptions if something goes wrong.

Reply:
yeah right, I'll change it